pkg/sanity/node.go

```diff
@@ -538,21 +516,7 @@ var _ = DescribeSanity("Node Service", func(sc *TestContext) {
 
 			vol := r.MustCreateVolume(
 				context.Background(),
-				&csi.CreateVolumeRequest{
-					Name: name,
-					VolumeCapabilities: []*csi.VolumeCapability{
-						{
-							AccessType: &csi.VolumeCapability_Mount{
-								Mount: &csi.VolumeCapability_MountVolume{},
-							},
```

Comment:
This is not present in `MakeCreateVolumeReq` result. Should it be there?

Reply:
Yes, `TestVolumeCapabilityWithAccessType` is also used in `MakeCreateVolumeReq`